samples/nrf9160/download/README.rst

```diff
@@ -0,0 +1,129 @@
+.. _download_sample:
+
+nRF9160: Download sample
+########################
+
+The Download sample demonstrates how to download a file from a HTTP or CoAP server using the download_client library, with optional TLS or DTLS.
+
+Overview
+********
+
+The sample first initializes the :ref:`nrfxlib:bsdlib` and AT communications.
+Next, it provisions a certificate to the modem using the :ref:`modem_key_mgmt` library if :option:`CONFIG_SAMPLE_SECURE_SOCKET` is set.
+The certificate file name and security tag can be configured via :option:`CONFIG_SAMPLE_SEC_TAG` and :option:`CONFIG_SAMPLE_CERT_FILE`, respectively
```

Comment:
nit: Dot missing at the end of the sentence.

Reply:
Fixed.